src/main/java/model/Wallet.java

```diff
@@ -0,0 +1,60 @@
+package model;
+
+import exception.InvalidMoneyAmountException;
+
+public class Wallet {
+    private int money;
+
+    /**
+     * @param money: integer
+     * @pre money >= 0
+     */
+    public Wallet(int money) {
+        this.money = Math.max(money, 0);
+    }
+
+    public Wallet() {
```

Comment:
To me, having this default is not intuitive, as always (apart from testing), the value is set directly